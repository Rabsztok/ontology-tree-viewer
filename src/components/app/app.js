import React from 'react'
import Route from 'react-router-dom/Route'
import Switch from 'react-router-dom/Switch'
import Tree from 'pages/index_page/index_page'
import { injectGlobal, ThemeProvider } from 'styled-components'
import theme from 'theme'

injectGlobal`
  * { box-sizing: border-box; }
`

/** Main container of the app, evaluated both on server and client run */
const App = () => (
  <ThemeProvider theme={theme}>
    <Switch>
      <Route path='/' component={Tree} />
    </Switch>
  </ThemeProvider>
)

export { App }
export default App
