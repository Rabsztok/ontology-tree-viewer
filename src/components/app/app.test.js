import React from 'react'
import { shallow } from 'enzyme'
import { App } from './app'

describe('App', () => {
  it('renders without errors', () => {
    const component = shallow(<App />)
    expect(component.dive()).toMatchSnapshot()
  })
})
