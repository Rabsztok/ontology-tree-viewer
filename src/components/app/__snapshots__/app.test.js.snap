// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without errors 1`] = `
<Switch>
  <Route
    component={[Function]}
    path="/"
  />
</Switch>
`;
